kolibri/core/assets/src/views/CorePage/AppBarPage.vue

```diff
@@ -20,6 +20,9 @@
         type="indeterminate"
         :delay="false"
       />
+      <div aria-live="polite">
+        <StorageNotification />
```

Comment:
Nice! This looks good from the code side. Let's remember to have @radinamatic do some testing on this once we get the backend hooked up to verify!